tools/sim/bridge.py

```diff
@@ -64,12 +66,16 @@ def steer_rate_limit(old, new):
 
 class Camerad:
   def __init__(self):
-    self.frame_id = 0
+    self.frame_road_id = 0
+    self.frame_wide_id = 0
     self.vipc_server = VisionIpcServer("camerad")
 
     # TODO: remove RGB buffers once the last RGB vipc subscriber is removed
     self.vipc_server.create_buffers(VisionStreamType.VISION_STREAM_RGB_ROAD, 4, True, W, H)
     self.vipc_server.create_buffers(VisionStreamType.VISION_STREAM_ROAD, 40, False, W, H)
+
+    self.vipc_server.create_buffers(VisionStreamType.VISION_STREAM_RGB_WIDE_ROAD, 4, True, W, H)
+    self.vipc_server.create_buffers(VisionStreamType.VISION_STREAM_WIDE_ROAD, 40, False, W, H)
```

Comment:
Creating all these buffers probably allocates a ton op RAM on the GPU, causing issues on lower-end end hardware. Can you try allocating 5 for both fcam and ecam instead of 40? We use more in openpilot to ensure loggerd doesn't drop frames, but we don't care with the simulator.

Reply:
Buffer of 5 seems to work.